website/pages/en/index.js

```diff
@@ -148,38 +134,108 @@ const GetStartedButton = ({ href }) => (
 )
 
 const BuildingBlocks = props => (
-  <BuildingBlocksContainer>
-    <EmptyStateCard
-      title="aragonOS"
-      text="Solidity framework for governance"
+  <BuildingBlocksContainer className="white-section">
+    <BlocksCards
+      title="AragonOS"
+      text={<p>Solidity framework for governance</p>}
       actionButton={() => <GetStartedButton href={docUrl('aragonos-intro')}/>}
-      icon={() => <IconSettings /> }
+      icon={() => <img src={siteConfig.baseUrl + siteConfig.os} />}
     />
-    <EmptyStateCard
-      title="aragonAPI"
-      text="Easily interact with your dapp's state"
+    <BlocksCards
+      title="AragonAPI"
+      text={<p>Easily interact with your dapp's state</p>}
       actionButton={() => <GetStartedButton href={docUrl('aragonjs-intro')}/>}
-      icon={() => <IconTokens /> }
+      icon={() => <img src={siteConfig.baseUrl + siteConfig.api} />}
     />
-    <EmptyStateCard
-      title="aragonUI"
-      text="Create a beautiful UI for your dapp"
+    <BlocksCards
+      title="AragonUI"
+      text={<p>Create a beautiful UI for your dapp</p>}
       actionButton={() => <GetStartedButton href={docUrl('aragonui-intro')}/>}
-      icon={() => <IconBlank /> }
+      icon={() => <img src={siteConfig.baseUrl + siteConfig.ui} />}
     />
   </BuildingBlocksContainer>
 )
 
-const Intro = props => (
-  <div className="productShowcaseSection paddingTop">
-    <h2>Build DAOs, protocols and dapps</h2>
-    <MarkdownBlock>
-      Aragon is the most **powerful** and **modular** way to run [DAOs](https://en.wikipedia.org/wiki/Decentralized_autonomous_organization).
-      But on our journey creating DAOs, we discovered how to modularize individual components that can be used for creating **any dapp or crypto protocol**.
-    </MarkdownBlock>
+const BuildingBlocksContainer = styled.div`
+  display: flex;
+  align-items: center;
+  justify-content: center;
+  padding: 2rem 1rem;
+  flex-wrap: wrap;
+`
+
+const BlocksCards = styled(EmptyStateCard)`
+  border-radius: 6px;
+  box-shadow: 0 4px 6px 0 rgba(0, 0, 0, 0.08);
+  background-color: #f9fafc;
+  border: 0px solid transparent;
```

Comment:
Is this because of the border moving the layout when setting/unsetting? A lot of things may get modified but I think `* { box-sizing: border-box; }` will help out with this.